app/[locale]/blog/[slug]/page.tsx

```
@@ -342,9 +482,18 @@
                 <span className="text-secondary-600 dark:text-secondary-400">
                   {t('post.share')}
                 </span>
-                <button className="bg-white dark:bg-secondary-800 p-2 rounded-lg shadow hover:shadow-md transition-shadow border border-secondary-200 dark:border-secondary-700">
+                <button
+                  onClick={handleShare}
+                  className="bg-white dark:bg-secondary-800 p-2 rounded-lg shadow hover:shadow-md transition-shadow border border-secondary-200 dark:border-secondary-700 hover:bg-primary-50 dark:hover:bg-primary-900/20"
+                  title={t('post.sharePost')}
+                >
                   <Share2 className="w-4 h-4 text-secondary-600 dark:text-secondary-400" />
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)